common/src/dodeca.rs

```diff
@@ -306,4 +346,26 @@ mod tests {
             epsilon = 1e-10
         );
     }
+
+    #[test]
+    fn chunk_to_node() {
```

Comment:
Leaving this comment for posterity:

Before the rebase, instead of these two tests, there was only one test: `dual_to_chunk_factor_correct()` defined as follows:
```rust
// The center of a dodeca should have chunk coordinates of (1, 1, 1)
let corner = Vertex::A.node_to_dual() * math::origin();
assert_abs_diff_eq!(
    Vertex::dual_to_chunk_factor(),
    1.0 / (corner.x / corner.w),
    epsilon = 1e-10
);
```
This test is no longer in the commit history. It was removed because the `chunk_to_node` test is just as effective at catching bugs but is easier to understand.